src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -348,6 +365,18 @@ public function testThrowExceptionIfFormatDoesNotContainYearMonthAndDay()
 
     /**
      * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @expectedExceptionMessage The "format" option should contain the letters "y", "M" or "d". Its current value is "wrong".
+     */
+    public function testThrowExceptionIfFormatDoesNotContainYearMonthOrDay()
```

Comment:
I would include the fact that this is testing the specific behaviour for the `single_text` widget (something like `testThrowExceptionIfFormatMissesYearMonthAndDayWithSingleTextWidget()`).